tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -846,6 +846,40 @@ def predict(self, context, model_input):
     assert loaded_model.predict(iris_data[0])
 
 
+@pytest.mark.large
+def test_multiple_artifacts_with_common_sub_paths_resolve_correctly_when_saving_model(
+    sklearn_knn_model, model_path, iris_data
+):
+    default_model_path = "model"
+    with mlflow.start_run():
+        mlflow.sklearn.log_model(sk_model=sklearn_knn_model, artifact_path=default_model_path)
+        first_model_uri = "runs:/{run_id}/{artifact_path}".format(
+            run_id=mlflow.active_run().info.run_id, artifact_path=default_model_path
+        )
```

Comment:
```suggestion
        model_info = mlflow.sklearn.log_model(sk_model=sklearn_knn_model, artifact_path=default_model_path)
        first_model_uri = model_info.model_uri
```

This block can be simplified by using the return value of `log_model`.